tests/ZendTest/Mvc/Controller/Plugin/FilePostRedirectGetTest.php

```diff
@@ -274,59 +274,6 @@ public function testReuseMatchedParametersWithSegmentController()
         $this->assertEquals(303, $prgResultRoute->getStatusCode());
     }
 
-    public function testFieldsetAmountInFormEqualsFieldsetsInInputFilter()
```

Comment:
I deleted this test, because now a `CollectionInputFilter` is used for collections and there is no need for checking the same amount of fieldsets and `InputFilter`s. Please correct me, if I am wrong.